docs/topics/certificaterotation.md

```diff
@@ -0,0 +1,52 @@
+# Cluster certificate rotation
+
+Instructions on rotating TLS CA and certificates for an AKS Engine cluster.
+
+## Prerequesites
+
+- Kubernetes version 1.8.0 or later is required
+- The etcd members MUST be in a healthy state before rotating the CA and certs.
+- The `_output` directory for your cluster deployment, including a working SSH key for accessing the master nodes and `kubeconfig`.
+
+## Preparation
+
+**CAUTION**: Before rotation, it's preferrable to back up your cluster:
+
+1. On an etcd node, take a backup of the current state.
+2. //TODO recommend etcd backup tool, extract the existing state from etcd. Copy this back to your working machine as a precaution, in case the control plane goes down.
+3. Keep a copy of your current _output directory.
```

Comment:
Not sure what the distinction is between this guidance and the prerequisite guidance above.

Reply:
true, there isn't one. Will remove one of them.